docusaurus/docs/create-a-plugin/_category_.json

```diff
@@ -1,6 +1,6 @@
 {
     "position": 3,
-    "label": "Development",
+    "label": "Create a plugin",
```

Comment:
I find it quite confusing to have a Create a plugin section then have Create a plugin page under Get Started

Reply:
WDYT @jackw ?

Reply:
What if we merged the content of both docs/getting-started/ and docs/getting-started/creating-a-plugin? There is already some overlap and I'm not sure that it makes sense to keep these topics separate at this point.

Reply:
Sure - I'm happy enough with that suggestion 👍

Reply:
Done. creating-a-plugin.mdx is deleted, content incorporated into get-started.mdx